### PR TITLE
Feature/tune 90 central cms yourwords

### PR DESCRIPTION
**Pull request by RichardLanders**

### CODE REVIEW

**Comment by RichardLanders** [comment]
@saeedmb8 @miltontony

File: molo/yourwords/tests/test_wagtail_admin_action.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        )
+        client = Client()
+        client.login(username='superuser', password='pass')
+
+        response = self.client.post('/admin/modeladmin/yourwords/'
+                                    'yourwordscompetitionentry/')
+
+        date = str(datetime.datetime.now().date())
+
+        print response
```
Comment (miltontony) [inline molo/yourwords/tests/test_wagtail_admin_action.py]: stray print

File: molo/yourwords/tests/test_wagtail_admin_action.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+        expected_output = (
+            'competition,submission_date,user,story_name,story_text,'
+            'terms_or_conditions_approved,hide_real_name,is_read,'
+            'is_shortlisted,is_winner\r\n'
+            '7,{0},1,test,test body,1,1,0,0,0'.format(date)
+        )
+
+        print expected_output
```
Comment (miltontony) [inline molo/yourwords/tests/test_wagtail_admin_action.py]: stray print

File: molo/yourwords/wagtail_hooks.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    model = YourWordsCompetitionEntry
+    menu_label = 'Entries'
+    menu_icon = 'edit'
+    index_view_class = ModelAdminTemplate
+    add_to_settings_menu = False
+    list_display = ['story_name', 'user', 'hide_real_name',
+                    'submission_date', 'is_read', 'is_shortlisted',
+                    'is_winner', '_convert']
+
+    list_filter = ['competition__slug']
```
Comment (smarzban) [inline molo/yourwords/wagtail_hooks.py]: Can we filter here with ['submission_date', 'is_read', 'is_shortlisted', 'is_winner'] instead

File: molo/yourwords/templates/admin/yourwords/yourwords_date_range_filter.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    <p class="submit-row">
+        {#create hidden inputs to preserve values from other filters and search field#}
+        {% for k, v in i.get_query.items %}
+                <input type="hidden" name="{{ k }}" value="{{ v }}">
+        {% endfor %}
+    <button type="submit">{% trans "Search" %}</button>
+    <button type="submit" id="resetbtn">{% trans "Clear" %}</button>
+    </p>
+</form>
+{% endwith %}
```
Comment (g1039) [inline molo/yourwords/templates/admin/yourwords/yourwords_date_range_filter.html]: Missing a blank line at the of the file.

File: molo/yourwords/models.py [hunk 5]
```diff
@@ -117,6 +117,23 @@ class YourWordsCompetitionEntry(models.Model):
         help_text=_('Page to which the entry was converted to')
     )
 
+    panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel('competition'),
+                # FieldPanel('submission_date'),
```
Comment (smarzban) [inline molo/yourwords/models.py]: @nathanbegbie why is this commented out?

**Comment by smarzban** [comment]
👍  🍰🍰🍰🍰🍰🍰🍰

**Pull request merged**